source/com/dub.d

```diff
@@ -210,6 +210,41 @@ bool setConfiguration(string configuration)
 	return updateImportPaths(false);
 }
 
+/// List all possible arch types, even one's not possible for every compiler (eg ARM for dmd).
+/// Call_With: `{"subcmd": "list:arch-types"}`
+@arguments("subcmd", "list:arch-types")
+string[] archTypes() @property
+{
+	return [ "x86", "x86_64" ]; // todo update this list to include all other archs
```

Comment:
gdc also has `arm` and `arm_thumb`, can you also add those when `compilerName.toLower.contains("gdc")`?

https://github.com/dlang/dub/blob/8f87d9f8413d9e962331accd26d7ccee9ab69b35/source/dub/compilers/gdc.d#L70-L73